src/ingestion_data.py

```diff
@@ -4,6 +4,9 @@
 
 from src.graph_model import (Version, EventType, SecurityEvent, FeedBackType, StatusType, EcosystemType)
 from src.parse_datetime import from_date_str, get_date, get_year, get_yearmonth
+from src.config import MAX_STRING_LENGTH
+
+from src.utils import get_cves_from_text
```

Comment:
I suggest to keep `get_cves_from_text` local to this package as I don't anticipate any use outside of this.

Reply:
moved to same file